### ISSUE TITLE
Support for C++

### ISSUE THREAD

**Issue by adimishra1** [whitequark/rainbowth#32]
Does it support C++?

**Comment by paramjeetdhiman** [comment]
yeah it does support c++ and other languages I have attached my config file here just paste it in below location

**sublime-text/User/Rainbowth.sublime-settings**
`{
    "languages": ["c++", "c", "python","lisp", "scheme", "clojure", "clojurescript", "hylang","js","sass","scss"],
    "palettes": {
        "default": ["#0E4A8E","#3F9101","#BCBF01","#BC0BA2","#61AA0D","#3D017A","#D6A60A","#7710A3","#A502CE","#eb5a00"],
        "Dracula": ["#E8BA36","#54A857","#359FF4","#5060BB","#179387","#A5BE00","#005FA3","#DB7100","#FFC666","#38FF91"],
        "One Dark": ["#359FF4","#E8BA36","#54A857","#5060BB","#179387","#A5BE00","#005FA3","#DB7100","#FFC666","#38FF91"]
    },
    "disable_inside_comment": true,
    "disable_inside_string": true, 
    "custom_signs":
    {
        "enabled": true,
        "prefix": "({[",
        "suffix": ")}]"
    }
}`
